docs/reference/checks_config.md

```diff
@@ -0,0 +1,183 @@
+            # ... comparison config
+        }
+    ]
+}
+```
+
+### 4. List Expression
+
+Validates lists of items using one of three operators:
+
+- `exists` - Checks if the list exists and is not empty
+- `any` - At least one item must match the predicate
+- `all` - All items must match the predicate
+
+```python
+{
+    "type": "list",
+    "operator": "any",  # "any", "all", or "exists"
+    "path": "metadata.creators",  # Path to the list field
+    "predicate": {  # Condition to apply to list items (required for "any" and "all")
+        "type": "expression_type",
+        # ... expression configuration
+    }
+}
+```
+
+## Complete Example
+
+Here's a complete example showing multiple rules with different expression types:
+
+```python
+{
+    "id": "metadata-validation",
+    "title": "Metadata Validation",
+    "description": "Validates basic metadata requirements",
+    "rules": [
+        {
+            "id": "license:exists",
+            "level": "error",
+            "title": "Record license",
+            "checks": [
+                {
+                    "path": "metadata.rights",
+                    "type": "list",
+                    "operator": "exists",
+                    "predicate": {}
+                }
+            ],
+            "message": "Licenses are required.",
+            "description": "All submissions must specify the licensing terms."
+        },
+        {
+            "id": "creators:identifier",
+            "level": "info",
+            "title": "Creator Identifiers",
+            "checks": [
+                {
+                    "path": "metadata.creators",
+                    "type": "list",
+                    "operator": "all",
+                    "predicate": {
+                        "type": "logical",
+                        "operator": "and",
+                        "expressions": [
+                            {
+                                "path": "person_or_org.identifiers",
+                                "type": "field"
+                            },
+                            {
+                                "path": "person_or_org.identifiers",
+                                "type": "list",
+                                "operator": "any",
+                                "predicate": {
+                                    "path": "identifier",
+                                    "type": "field"
+                                }
+                            }
+                        ]
+                    }
+                }
+            ],
+            "message": "Affiliations are recommended for all creators (e.g. an ORCID)",
+            "condition": {
+                "path": "metadata.creators",
+                "type": "list",
+                "operator": "exists",
+                "predicate": {
+                }
+            },
+            "description": "All creators should have a persistent identifier (e.g. an ORCID)"
+        },
+    ]
+}
+```
```

Comment:
Here, I would also expect an example of what kind of result comes back from evaluating the above rules against a record.